common/data_refinery_common/job_management.py

```diff
@@ -256,13 +195,21 @@ def create_processor_job_for_original_files(original_files: List[OriginalFile],
         processor_job.save()
 
         for original_file in original_files:
+            if original_file.is_blacklisted():
+                logger.debug("Original file had a blacklisted extension of %s, skipping",
+                            extension=original_file.get_extension(),
+                            original_file=original_file.id)
+                original_file.delete_local_file()
+                continue
+
             assoc = ProcessorJobOriginalFileAssociation()
             assoc.original_file = original_file
             assoc.processor_job = processor_job
             assoc.save()
 
         logger.debug("Queuing processor job.",
-                     processor_job=processor_job.id)
+                     processor_job=processor_job.id,
+                     downloader_job=downloader_job.id)
```

Comment:
Note that the `downloader_job` parameter defaults to None, this could cause an error.